src/lib/apis/attachment.ts

```
@@ -11,66 +11,45 @@ export class MaximAttachmentAPI extends MaximAPI {
-					reject(error);
-				});
-		});
+			);
+
+			if ("error" in response) {
+				throw response.error;
+			}
+
+			return response.data;
+		} catch (error) {
+			throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)